website/content/docs/runner/index.mdx

```diff
@@ -37,6 +37,102 @@ to scale further.
 
 On-demand runners: see [on-demand runners](/docs/runner/on-demand-runner) for more information.
 
+## Runner Targeting
+
+Runner targeting allows Waypoint apps to choose on which runners their operations happen. A runner `profile` can be specified
+in the `runner` stanza in the `waypoint.hcl` file. The `runner profile` can be configured to use a specific static runner,
+or multiple, based on labels.
+
+### Runner Targeting Example
+
+Runners can have labels applied to them with the `-label` flag provided alongside the `waypoint runner agent` command.
+
+```terminal
+% WAYPOINT_SERVER_ADDR=localhost:9701 WAYPOINT_SERVER_TLS=true waypoint runner agent -label=org=hashicorp -label=env=dev -cookie=0000000-0000-0000-0000-000000000000
```

Comment:
```suggestion
$ waypoint runner agent -label=org=hashicorp -label=env=dev
```

I'm not sure if cookie is required, but let's simplify this command so people can see the important flags for labeling a runner